client/gutenberg/extensions/slideshow/style.scss

```diff
@@ -8,9 +7,11 @@
 		line-height: normal;
 	}
 
-	// Override UL left margin from theme styles
-	.edit-post-visual-editor & ul.wp-block-jetpack-slideshow_swiper-wrappper {
+	// Override <ul /> and <li /> left margins and paddings from theme styles
+	.edit-post-visual-editor & ul.wp-block-jetpack-slideshow_swiper-wrappper,
+	.edit-post-visual-editor & li.wp-block-jetpack-slideshow_slide {
```

Comment:
The repitition of `edit-post-visual-editor &` is a bit ugly, it could be nested.

```sass
.edit-post-visual-editor & {
	ul.wp-block-jetpack-slideshow_swiper-wrappper,
	li.wp-block-jetpack-slideshow_slide {
		// …
	}
}
```